opus/application/static_media/js/browse.js

```diff
@@ -1619,7 +1651,10 @@ var o_browse = {
         let tab = opus.getViewTab(view);
         let footerHeight = $(".app-footer").outerHeight();
         let mainNavHeight = $("#op-main-nav").outerHeight();
-        let navbarHeight = $(".panel-heading").outerHeight();
+        // we need to specify the tab here
```

Comment:
Do we need this comment? We select using tab all over the place without explaining why.